pyseir/inference/model_fitter.py

```diff
@@ -240,36 +240,47 @@ def set_inference_parameters(self):
             self.fit_params["hosp_fraction"] = 1
 
         if len(self.fips) == 5:
-            OBSERVED_NEW_CASES_GUESS_THRESHOLD = 2
-            idx_enough_cases = np.argwhere(
-                np.cumsum(self.observed_new_cases) >= OBSERVED_NEW_CASES_GUESS_THRESHOLD
-            )[0][0]
-            initial_cases_guess = np.cumsum(self.observed_new_cases)[idx_enough_cases]
-            t0_guess = list(self.times)[idx_enough_cases]
-
             state_fit_result = load_inference_result(fips=self.state_obj.fips)
-            self.fit_params["t0"] = t0_guess
+            T0_LEFT_PAD = 5
+            T0_RIGHT_PAD = 30
```

Comment:
This is off in the weeds and not blocking, but I think 30 might be too large of a guess. Also, I totally own that I set it to 30 initially.

Reply:
Agreed, but I sensitivity analysis is out of scope here. I am just trying to reinstate the 297 counties we weren't going to show model results.